src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2018 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+'use strict';
+
+// Polyfill in Firefox.
+// See https://blog.mozilla.org/webrtc/getdisplaymedia-now-available-in-adapter-js/
+if (adapter.browserDetails.browser == 'firefox') {
+  adapter.browserShim.shimGetDisplayMedia(window, 'screen');
+}
+
+function handleSuccess(stream) {
+  const video = document.querySelector('video');
+  video.srcObject = stream;
+}
+
+function handleError(error) {
+  errorMsg(`getDisplayMedia error: ${error.name}`, error);
+}
+
+function errorMsg(msg, error) {
+  const errorElement = document.querySelector('#errorMsg');
+  errorElement.innerHTML += `<p>${msg}</p>`;
+  if (typeof error !== 'undefined') {
+    console.error(error);
+  }
+}
+
+if ('getDisplayMedia' in navigator) {
+  navigator.getDisplayMedia({video: true})
```

Comment:
Use async/await instead as we're changing to that in all other samples.

Reply:
`await is only valid in async functions and async generators`
-- Firefox doesn't like await outside functions yet.

Reply:
oh... neither does chrome and @jan-ivar tells me this will never work :-)

Reply:
How about;
```
async function init(config) {
  try {
    const stream = await navigator.getDisplayMedia(config);
    handleSucces(stream);
  } catch (e) {
    handleError(e);
  }
}
init(config);
```

Reply:
i'd rather keep the style consistent with the other samples here and then do a big pass for async-await at another point.

Reply:
Sounds good!